Makefile

```diff
@@ -770,20 +770,28 @@ install.docker-full: install.docker install.docker-docs
 
 .PHONY: install.systemd
 ifneq (,$(findstring systemd,$(BUILDTAGS)))
+
+define install-systemd-unit
+	install ${SELINUXOPT} -m 644 $(1) $(2)
+	if [ -n "$(3)" ]; then \
```

Comment:
Why? I mean, wouldn't it just be simpler to always replace?

With a conditional, a human has to keep track of which files have `@@PODMAN@@`, then, if it gets added to a new one, remember to add `, replace` to this magic invocation in an out-of-the-way place.

Reply:
I interpreted earlier comments to make the replacing more explicit.  I gladly simplify :+1:

Reply:
Better?